packages/feedback-service/src/resolver.ts

```diff
@@ -21,7 +22,27 @@ export const FeedbackResolver = {
   Mutation: {
     addFeedback(root: any, args: any, ctx: any) {
       const data = new Feedback(args.input);
-      return data.save();
+      return data.save().then(response => {
+        Feedback.findById(response._id)
+        .then( fb =>{
+          if (fb){
+            const issue = {
+              "fields":{
+                "project":{
+                  "key": "ONEPLAT"
+                },
+                "summary": fb.title,
+                "description": fb.description,
+                "issuetype": {"name": "Task"},
+              }
+            }
+            const envpath = process.env.NODE_ENV;
+            if(envpath === "production"){
+              addFeedback(issue);
```

Comment:
After the JIRA ticket is created, you need to store the ticketID in the `ticketID` field in the feedback schema.

Reply:
Can you please clear it ?
when i create ticket in jira it return something like bellow 
{ id: '774437',
 key: 'ONEPLAT-49',
...
}

Do i need to store key or id ?

Reply:
Updated code and tested.

Reply:
The key, as that is human understandable, and can also be used to create a link to the actual JIRA issue.

Reply:
Yes correct.